test/source/browser/browser-pool.ts

```diff
@@ -28,14 +26,13 @@ export class BrowserPool {
 
   public newBrowserHandle = async (t: AvaContext, closeInitialPage = true) => {
     await this.semaphore.acquire();
-    // ext frames in gmail: https://github.com/GoogleChrome/puppeteer/issues/2506 https://github.com/GoogleChrome/puppeteer/issues/2548
+    const extensionDir = t.extensionDir ?? this.extensionBuildDir;
     const args = [
       '--no-sandbox', // make it work in travis-ci
       '--disable-setuid-sandbox',
       '--kiosk-printing',
-      '--disable-features=site-per-process',
```

Comment:
If they actually fixed this `site-per-process` issue, that would mean we should be able to test the iframes directly where they are, for example, in Live tests we should be able to test them directly from the Gmail page, instead of copying the URL and opening a separate tab, and testing there. That would speed up the tests. Let's file that for another issue (unless you have already done exactly that).

Reply:
Yes, it seems issue was fixed as it's closed now in puppeteer repo - https://github.com/puppeteer/puppeteer/issues/2548

I haven't changed test logic yet, just checked that all tests pass without `--disable-features=site-per-process` flag